projects/components/src/datagrid/datagrid.component.spec.ts

```diff
@@ -674,6 +752,14 @@ export class HostWithDatagridComponent {
 
     paginationText = 'Total Items';
 
+    pagination: {
```

Comment:
Please create a definition for it, explaining what each thing means. Let's make it all lower case so it doesn't look like a string that could be displayed. Didn't we say we would have a way for the user to specify the height of each row?

Reply:
I think I did what you were asking for here, let me know if it isn't. Also, totally forgot about the row height - added that in the latest commit (and a test for it).